HandleNHNotificationCall/index.ts

```diff
@@ -24,9 +34,37 @@ export async function index(
   context: Context,
   notificationHubMessage: NotificationHubMessage
 ): Promise<void> {
-  await df
-    .getClient(context)
-    .startNew("HandleNHNotificationCallOrchestrator", undefined, {
-      message: notificationHubMessage
-    });
+  switch (notificationHubMessage.kind) {
+    case DeleteKind.DeleteInstallation:
+      const client = df.getClient(context);

```

Comment:
You can create the client outside of the case, so you don't have the problem of scoping the same variable in different cases. 
Rule of thumb: try not to instantiate variables inside switch cases (I thought we had [this rule ](https://eslint.org/docs/rules/no-case-declarations)applied).

Also, consider using fall-through when multiple cases share the same logic:
```ts
  const client = df.getClient(context);
  switch (notificationHubMessage.kind) {
    case DeleteKind.DeleteInstallation:
    case CreateOrUpdateKind.CreateOrUpdateInstallation:
    case NotifyKind.Notify:
      await client.startNew(
        "HandleNHNotificationCallOrchestrator",
        undefined,
        {
          message: notificationHubMessage
        }
      );
      break;
    default:
      assertNever(notificationHubMessage);
      break;
  }
```

Reply:
> Also, consider using fall-through when multiple cases share the same logic:

This is in preparation of having different code within each `case`. 
I thought It would be simpler both for me and you to manage the new calls in this way.